### PR TITLE
Add OverSight v1.0.0

### PR DESCRIPTION
**Pull request by tennisonchan**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/oversight.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'oversight' do
+  version '1.0.0'
+  sha256 'e20b7d91f4301f26236607e5c204286b1901cd138ae6c3a66865c09a95746bc5'
+
+  # bitbucket.org was verified as official when first introduced to the cask
+  url 'https://bitbucket.org/objective-see/deploy/downloads/OverSight_1.0.0.zip'
```
Comment (fanquake) [inline Casks/oversight.rb]: Use #{version} in the URL stanza

File: Casks/oversight.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name 'Oversight'
+  homepage 'https://objective-see.com/products/oversight.html'
+
+  app 'OverSight_Installer.app'
+
+  uninstall delete: '/Applications/OverSight.app'
```
Comment (vitorgalvao) [inline Casks/oversight.rb]: Are you sure this is the only thing the installer does? That’s unlikely. Doesn’t objective-see usually tell exactly how to uninstall their apps?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/oversight.rb (hunk 2 above)
Comment (adidalal) [inline Casks/oversight.rb]: Needs to be something like:

```
uninstall quit: {{package_id_here}}
          delete: '/Applications/OverSight.app'
          login_item: 'Oversight'
```

File: Casks/oversight.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'oversight' do
+  version '1.0.0'
+  sha256 'e20b7d91f4301f26236607e5c204286b1901cd138ae6c3a66865c09a95746bc5'
+
+  # bitbucket.org/objective-see was verified as official when first introduced to the cask
+  url 'https://bitbucket.org/objective-see/deploy/downloads/OverSight_#{version}.zip'
```
Comment (adidalal) [inline Casks/oversight.rb]: Mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

URL is: https://objective-see.com/products/versions/oversight.json

**Review by adidalal — CHANGES_REQUESTED** [review]

File: Casks/oversight.rb (hunk 2 above)
Comment (tennisonchan) [inline Casks/oversight.rb]: Not sure how to get the `{{package_id}}`.

Comment (adidalal) [inline Casks/oversight.rb]: Updated comment.
For future reference: http://superuser.com/questions/346369/getting-the-bundle-identifier-of-an-os-x-application-in-a-shell-script

File: Casks/oversight.rb [hunk 4]
```diff
@@ -3,11 +3,15 @@
...
   # bitbucket.org/objective-see was verified as official when first introduced to the cask
-  url 'https://bitbucket.org/objective-see/deploy/downloads/OverSight_#{version}.zip'
+  url "https://bitbucket.org/objective-see/deploy/downloads/OverSight_#{version}.zip"
+  appcast 'https://objective-see.com/products/versions/oversight.json'
   name 'Oversight'
   homepage 'https://objective-see.com/products/oversight.html'
 
-  app 'OverSight_Installer.app'
+  installer manual: 'OverSight_Installer.app'
+  app 'OverSight.app'
```
Comment (vitorgalvao) [inline Casks/oversight.rb]: This is to be used only when the app comes in the zip (not the case here).

File: Casks/oversight.rb [hunk 5]
```diff
@@ -3,11 +3,15 @@
   sha256 'e20b7d91f4301f26236607e5c204286b1901cd138ae6c3a66865c09a95746bc5'
 
   # bitbucket.org/objective-see was verified as official when first introduced to the cask
-  url 'https://bitbucket.org/objective-see/deploy/downloads/OverSight_#{version}.zip'
+  url "https://bitbucket.org/objective-see/deploy/downloads/OverSight_#{version}.zip"
+  appcast 'https://objective-see.com/products/versions/oversight.json'
```
Comment (vitorgalvao) [inline Casks/oversight.rb]: It needs a checkpoint. See the [docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by tennisonchan** [comment]
The file `usr/local/Caskroom/oversight/1.0.0/OverSight_Installer.app` is not found. How can I modify the path of the installer? 

screen shot 2016-10-16 at 6 47 01 pm

**Comment by vitorgalvao** [comment]
> The file `usr/local/Caskroom/oversight/1.0.0/OverSight_Installer.app` is not found. How can I modify the path of the installer?

Reread the message. It’s the other (wrong) stanza that is not being found.

**Review by vitorgalvao — APPROVED** [review]

**Review by adidalal — APPROVED** [review]

**Comment by adidalal** [comment]
Just needs a `brew cask style --fix {{cask_file}}`, looks like

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1910a740986bf8a0e62e9ca03502067c87221f63. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by tennisonchan** [comment]
Thank you so much for the modifications!

**Pull request closed without merging**